squid-blockexplorer/src/balances/processEvent.ts

```diff
@@ -24,8 +24,8 @@ export class BalanceEventHandler {
   getTransferAccounts(event: Event) {
     const data = new BalancesTransferEvent(this.ctx, event);
 
-    if (data.isV0) {
-      return [toHex(data.asV0.from), toHex(data.asV0.to)];
+    if (data.isV1) {
```

Comment:
What's the difference between `isV0` and `isV1`

Reply:
it's a runtime spec version - https://docs.substrate.io/maintain/runtime-upgrades/